src/initializers/koa/add-request-id.ts

```diff
@@ -1,8 +1,13 @@
 import { Context, Middleware } from 'koa';
+import { randomUUID } from 'crypto';
 
-export default function(config): Middleware {
+export default function (config): Middleware {
   return async function addRequestId(ctx: Context, next: () => void) {
-    const requestId = ctx.headers[config.traceHeaderName.toLowerCase()];
+    let requestId = ctx.headers[config.traceHeaderName.toLowerCase()];
+    if (!requestId) {
+      requestId = `orka-${randomUUID()}`;
```

Comment:
🙃 If we don't set the requestId here riviere will set one for the server logs and a different one for the http requests. Setting it here make sure that the same id is shared everywhere.